### PR TITLE
beginning to add data for 2023-06-06

### PR DESCRIPTION
**Pull request by tracykteal**
PR is a work in progress for week 23

### CODE REVIEW

**Comment by tracykteal** [comment]
Dataset now ready for review!

File: data/2023/2023-06-06/readme.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+# Read in with tidytuesdayR package 
+# Install from CRAN via: install.packages("tidytuesdayR")
+# This loads the readme and all the datasets for the week of interest
+
+# Either ISO-8601 date or year/week works!
+
+tuesdata <- tidytuesdayR::tt_load('2023-06-06')
+tuesdata <- tidytuesdayR::tt_load(2023, week = 23)
+
+owid-energy <- owid-energy.csv
```
Comment (jonthegeek) [inline data/2023/2023-06-06/readme.md:72]:
```suggestion
owid_energy <- tuesdata$`owid-energy`
```

File: data/2023/2023-06-06/readme.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+# Or read in the data manually
+
+owid-energy <- readr::read_csv('https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2023/2023-06-06/owid-energy.csv')
```
Comment (jonthegeek) [inline data/2023/2023-06-06/readme.md:76]:
```suggestion
owid_energy <- readr::read_csv('https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2023/2023-06-06/owid-energy.csv')
```

**Review by jonthegeek — CHANGES_REQUESTED** [review]
Just a couple tweaks to make things syntactic.

**Review by jonthegeek — APPROVED** [review]
Ok, that will do it, thanks!

**Pull request merged**